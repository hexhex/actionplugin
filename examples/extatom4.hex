% -*- prolog -*-

a(X) :- &testA[nil](X).
b(X) :- &testA[nil](X).

x(X, baz) :- a(X).
x(X, fubar) :- b(X).
