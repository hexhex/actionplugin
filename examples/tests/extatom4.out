{x(foo,fubar), x(foo,baz), b(foo), a(foo)}
